StyleGuide.mdown

```diff
@@ -0,0 +1,23 @@
+# Coding Styles
+
+## XSD Styles
+### Naming Styles
++ Attributes should be camelCased - The first letter of the second and subsequent words are capitalized.
++ Elements should be TitleCased (PascalCased) - The first letter of every word is capitalized.
++ Enumerations should be spinal-case - all lower case with spaces replaced with hyphens.
+
+### File Specific Styles
++ All files should be encoded in utf-8.
++ 4 Spaces should be used instead of tabs (or vice versa).
```

Comment:
Personally, just because the nesting can get out of control, I'd rather have two spaces as opposed to four (no...I'm not trying to start a flame war, just aim for easier readability on the web, etc...)